modules/sdk-coin-xlm/src/lib/keyPair.ts

```diff
@@ -0,0 +1,49 @@
+import * as stellar from 'stellar-sdk';
+import { DefaultKeys, Ed25519KeyPair, KeyPairOptions, NotImplementedError } from '@bitgo/sdk-core';
+
+export class KeyPair extends Ed25519KeyPair {
+  /**
+   * Public constructor. By default, creates a key pair with a random master seed.
+   *
+   * @param { KeyPairOptions } source Either a master seed, a private key, or a public key
+   */
+  constructor(source?: KeyPairOptions) {
+    super(source);
+  }
+
+  getKeys(raw = false): DefaultKeys {
+    const publicKey = stellar.StrKey.encodeEd25519PublicKey(Buffer.from(this.keyPair.pub, 'hex'));
+    const result: DefaultKeys = {
+      pub: raw ? stellar.StrKey.decodeEd25519PublicKey(publicKey).toString('hex') : publicKey,
+    };
+
+    if (this.keyPair.prv) {
+      const privateKey = stellar.StrKey.encodeEd25519SecretSeed(Buffer.from(this.keyPair.prv, 'hex'));
+      result.prv = raw ? stellar.StrKey.decodeEd25519SecretSeed(privateKey).toString('hex') : privateKey;
+    }
+
+    return result;
+  }
+
+  /** @inheritdoc */
+  getAddress(): string {
```

Comment:
Not sure if this is necessary for XLM

Reply:
I think its just the pub key in XLM format, so just return the pub